ext/cron/01_cron.ts

```diff
@@ -3,9 +3,73 @@
 // @ts-ignore internal api
 const core = Deno.core;
 
+function formateToCronSchedule(
+  value?: number | { exact: number | number[] } | {
+    start?: number;
+    end?: number;
+    every?: number;
+  },
+): string {
+  if (value === undefined) {
+    return "*";
+  } else if (typeof value === "number") {
+    return value.toString();
+  } else {
+    const { exact } = value as { exact: number | number[] };
+    if (exact === undefined) {
+      const { start, end, every } = value as {
+        start?: number;
+        end?: number;
+        every?: number;
+      };
+      if (start === undefined && end === undefined && every === undefined) {
+        return "*";
```

Comment:
I think this should throw with `TypeError("Invalid cron schedule")`

Reply:
Yes yes, Done.